src/java/test/org/apache/zookeeper/server/quorum/auth/QuorumAuthUpgradeTest.java

```diff
@@ -83,7 +83,7 @@ public void testNullAuthLearnerServer() throws Exception {
         Map<String, String> authConfigs = new HashMap<String, String>();
         authConfigs.put(QuorumAuth.QUORUM_SASL_AUTH_ENABLED, "false");
 
-        String connectStr = startQuorum(2, authConfigs, 0);
+        String connectStr = startQuorum(2, authConfigs, 0, false);
```

Comment:
Only those test cases which are extending `KerberosSecurityTestcase` class has the dependency with ApacheDS libraries. So, it is not required to add delay for other test cases.

 Internally, KerberosSecurityTestcase class is instantiating MiniKdc instance and ZK quorum auth module does handshaking using the credentials configured in `jaas.config` file.